src/puml.ts

```diff
@@ -60,13 +60,17 @@ class PlantUMLTask {
       "UTF-8",
     ]);
 
+    var svgData = '';
```

Comment:
I would suggest to use `let` instead of `var`

Reply:
Also, the TravisCI failed. It would be great if you could run prettier. Thank you!

Reply:
Also, what is the reason behind keeping `svgData` separate from `this.chunks`? It is still `this.chunks` that is used to parse the diagrams below.